slacktee.sh

```diff
@@ -77,7 +92,9 @@ function send_message()
 		found_pattern_prefix=""
 	fi
 
-	escaped_message=$(echo "$textWrapper\n$message\n$textWrapper" | sed 's/"/\\"/g' | sed "s/'/\\'/g" )
+	escaped_message=$(echo "$textWrapper\n$message\n$textWrapper" \
+		| sed 's/"/\\"/g' \
+		| sed "s/'/\\'/g")
```

Comment:
escaped_message is passed to 'curl' command as a parameter, so we need to escape " and ' properly. Otherwise, it causes an error of 'curl'.